af_data_adoxa_loader/models/calendar_appointment.py

```diff
@@ -139,9 +149,13 @@ def transform(self, row, transformations):
                 elif key == 'duration_selection':
                     if row['duration'] == "60":
                         row[key] = "1 hour"
-                    else:
+                    elif row['duration'] == "30":
                         row[key] = "30 minutes"
+                    else:
+                        create = False
                 elif key == 'type_id':
+                    if row[key] == "23":
+                        row[key] == "26"
```

Comment:
What does these numbers mean?
We should use some sort of constants for this, so that the code becomes readable and understandable.

Reply:
A constant would be clumsy due to translation, added clarifying comment instead